build.gradle

```diff
@@ -476,6 +477,19 @@ allprojects {
   tasks.eclipse.dependsOn(cleanEclipse, copyEclipseSettings)
 }
 
+// Set the system keystore/truststore password if we're running tests in a FIPS-140 JVM
+allprojects {
+  tasks.withType(RandomizedTestingTask) {
+    // So that this gets executed only right before the test runs
+    doFirst {
```

Comment:
This does not actually guarantee that it will be run right before the test is executed. Other doFirst blocks could (and are) added throughout the build, and which order they run in is dependent on when they are added to the task.

I think this would be better as setting a project property in `BuildPlugin.globalBuildInfo` alongside how we set `runtimeJavaHome`. Then in `BuildPlugin.commonTestConfig` have a condition on whether it is a fips jvm to add these sysprops.